web/src/app/chat/components/files/FilePicker.tsx

```diff
@@ -49,55 +52,88 @@ export function FilePickerContents({
     <>
       {recentFiles.length > 0 && (
         <>
-          <label className="text-sm font-light text-input-text p-2.5">
+          <Text text02 secondaryBody className="mx-2 mt-2 mb-1">
             Recent Files
-          </label>
+          </Text>
 
           {recentFiles.slice(0, 3).map((f) => (
-            <MenubarItem
+            <button
+              type="button"
```

Comment:
Hm, any reason why we're using raw `button`s instead of one of the provided button components?

Maybe the `LineItem` button would fit the bill here? It provides an interface to specify the leftIcon, title, and description.